synapse/res/providers.json

```diff
@@ -5,13 +5,11 @@
         "endpoints": [
             {
                 "schemes": [
-                    "https://twitter.com/*/status/*",
-                    "https://*.twitter.com/*/status/*",
                     "https://twitter.com/*/moments/*",
                     "https://*.twitter.com/*/moments/*"
```

Comment:
Weirdly these URLs don't seem to  be used by Twitter anymore? We can leave them for now, but it is a bit annoying they changed their URL scheme at some point...